bigbluebutton-html5/imports/ui/components/debug-window/component.jsx

```diff
@@ -0,0 +1,185 @@
+        hideBorder
+        contentLabel="Debug"
+      >
+        <div className={styles.row}>
+          <div className={styles.col} aria-hidden="true">
+            <h3 className={styles.title}>Debug</h3>
+          </div>
+        </div>
+
+        <div className={styles.row}>
+          <div className={cx(styles.col, styles.col_4)} aria-hidden="true">
+            <div className={styles.formElement}>
+              <div className={styles.label}>
+                User Agent:
+              </div>
+            </div>
+          </div>
+          <div className={styles.col}>
+            <div className={cx(styles.formElement, styles.pullContentRight)}>
+              <input id="debugModalUserAgent" type="text" readOnly value={window.navigator.userAgent} />
+            </div>
+          </div>
+          <div className={cx(styles.col, styles.col_2)}>
+            <div className={cx(styles.formElement, styles.pullContentRight)}>
+              <button
+                type="button"
+                onClick={() => navigator.clipboard.writeText(window.navigator.userAgent)}
+              >
+                Copy
+              </button>
+            </div>
+          </div>
+        </div>
+
+        <div className={styles.row}>
+          <div className={cx(styles.col, styles.col_4)} aria-hidden="true">
+            <div className={styles.formElement}>
+              <div className={styles.label}>
+                Enable Auto Arrange Layout:
+              </div>
+            </div>
+          </div>
+          <div className={cx(styles.col, styles.col_2)}>
+            <div className={cx(styles.formElement, styles.pullContentRight)}>
+              <Toggle
+                icons={false}
+                defaultChecked={autoArrangeLayout}
+                onChange={() => this.autoArrangeToggle()}
+                ariaLabel="teste"
+              />
+            </div>
+          </div>
+          <div className={styles.col}>
+            <div className={cx(styles.formElement, styles.pullContentRight)}>
+              <p className={styles.desc}>
+                (it will be disabled if you drag or resize the webcams area)
+              </p>
+            </div>
+          </div>
+        </div>
+      </Modal>,
+    );
+  }
+
+  render() {
+    const { dragging, position } = this.state;
+    if (!SHOW_DEBUG_WINDOW) return false;
+    return (
+      <Draggable
+        bounds="body"
+        onDrag={this.handleOnDrag}
+        onStop={this.handleDragStop}
+        onMouseDown={e => e.preventDefault()}
+        // disabled={swapLayout || isCameraFullscreen || BROWSER_ISMOBILE || resizing}
+        position={position}
+      >
+        <Button
+          className={styles.btn}
+          // icon="application"
+          customIcon={(
+            <i className={styles.debugIcon}>
+              <svg id="Icons" version="1.1" viewBox="0 0 32 32" xmlSpace="preserve" xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink">
```

Comment:
For consistency, we could probably have @tylercopeland create a debug icon and use the `Icon` component instead.